Lib/test/test_contextlib_async.py

```diff
@@ -11,14 +11,15 @@ def _async_test(func):
     """Decorator to turn an async function into a test case."""
     @functools.wraps(func)
     def wrapper(*args, **kwargs):
+        old_loop = asyncio.get_event_loop()
```

Comment:
Since asyncio.get_event_loop() API is weird, maybe an context manager helper should be added to test.support?